src/openforms/registrations/contrib/zgw_apis/plugin.py

```diff
@@ -444,3 +506,37 @@ def get_config_actions(self) -> list[Action]:
                 ),
             ),
         ]
+
+    def register_submission_to_objects_api(
+        self, submission: Submission, options: dict
+    ) -> dict:
+        object_mapping = {
+            "record.geometry": FieldConf(
+                RegistrationAttribute.locatie_coordinaat,
+                transform=_point_coordinate,
+            ),
+        }
+
+        context = {
+            "_submission": submission,
+            "productaanvraag_type": "ProductAanvraag",
+            "variables": get_variables_for_context(submission),
+            "submission": {
+                "public_reference": submission.public_registration_reference,
+                "kenmerk": str(submission.uuid),
+                "language_code": submission.language_code,
+            },
+        }
+
+        object_data = prepare_data_for_registration(
+            submission, context, options, object_mapping
+        )
+        response = send_data_to_api(object_data)
```

Comment:
see earlier remarks about this `send_data_to_api` helper - it's nicer to be consistent with the rest of the client implementation here and follow an OOP approach. That also makes it obvious where the Objects API client is coming from.